core/src/mindustry/mod/Mods.java

```diff
@@ -647,6 +646,106 @@ private void checkWarnings(){
                 cont.button("@ok", this::hide).size(300, 50);
             }}.show();
         }
+
+        //show list of missing dependencies
+        if(mods.contains(LoadedMod::hasUnmetDependencies)){
+            ui.loadfrag.hide();
+            new Dialog(""){{
+                setFillParent(true);
+                cont.margin(15);
+                cont.add("@mod.dependencies.error").colspan(2);
+                cont.row();
+                cont.image().width(300f).colspan(2).pad(2).height(4f).color(Color.scarlet);
+                cont.row();
+                cont.pane(p -> {
+                    mods.each(LoadedMod::hasUnmetDependencies, mod -> {
```

Comment:
How do I properly only check enabled mods for unmet dependencies? Since a mod with unmet dependencies is considered disabled, `m.enabled()` returns false. Should I just directly check the setting instead?